### DISCUSSION TITLE
The Blockchain db has become to big for my drive

### DISCUSSION THREAD

**Discussion by KaptainErk** [Chia-Network/chia-blockchain#8849] [Support]
I'm wondering if anyone could tell me how to move the blockchain_v1_mainnet.sqlite file to another drive as the one that it is on will no longer fit it?

I have some knowledge but a step by step process would be much appreciated. Using Windows 10 GUI by the way

Thanking you in advance

**Comment by wc357** [comment]
https://github.com/Chia-Network/chia-blockchain/discussions/7139
